services/openai_analysis_service/app.py

```diff
@@ -310,10 +310,20 @@ def run_openai_summarization(data_dict):
     result = {"summary": openai_summary}
 
     if USE_PINECONE:
-        e = client.embeddings.create(
-            input=openai_summary,
-            model=summarize_embedding_config.get("model", "text-embedding-ada-002"),
-        )
+        try:
+            e = client.embeddings.create(
+                input=openai_summary,
+                model=summarize_embedding_config.get("model", "text-embedding-3-small"),
```

Comment:
Can you pull the model name from the `embeddings_store` section of the config instead of hard-coding it?

Reply:
@profjsb That's already what this does. It defaults to `text-embedding-3-small` if no model is specified in the config, but this line here does read from `summarize_embedding_config` which contains the config's keys.

Reply:
@profjsb let me know when you have seen my answer, just so I know if you are fine with me merging as is